File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.py
class Solution(object):
    def removeStars(self, s):
        """
        :type s: str
        :rtype: str
        """
        res = []
        for c in s:
            if c != "*":
                res.append(c)
            else:
                if len(res) > 0:
                    res.pop()
        
        return "".join(res)
        

File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.go
func removeStars(s string) string {
    stack := []byte{}
    for _, v := range s {
        if byte(v) == '*' {
            stack = stack[:len(stack)-1]
        } else {
            stack = append(stack, byte(v))
        }
    }

    return string(stack)
}
